src/templates/base_register.html

```diff
@@ -79,15 +78,14 @@
     })
 });
 </script>
-
-{% endblock %}
-{% block extrajs %}
-{% endblock %}
```

Comment:
They are supposed to be added because you have committed those changes. You need to move your head to LH dirty's last commit and then send PR again. I am closing this one.